### ISSUE TITLE
[Suggestion] Link to Issues from CurseForge website

### ISSUE THREAD

**Issue by MarioSMB** [JoeFoxe/Hexerei-1.18#78]
Just a simple request to add a direct link to the Issues page alongside Source in the toolbar, thanks!

**Comment by PenGwenn** [comment]
I agree with this. I posted two bug reports in the comments because I was not able to find this page. I have since removed them from the comments, but I know it would benefit your players.

**Comment by MarioSMB** [comment]
Looks like this has since been resolved, thank you!
